war/src/main/webapp/css/responsive-grid.css

```diff
@@ -13,6 +13,12 @@
   -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
+h1.build-caption.page-headline {
+    max-width: 1200px;
```

Comment:
This does mean if they have a resolution higher than 1200px it'll still be cut off right?

Reply:
Well, I have checked that on viewport sized up to 1900px and everything worked fine. I can't say for sure what might happen for viewport sizes above 1900px